package.json

```diff
@@ -129,12 +147,10 @@
     "@web3-react/walletlink-connector": "^6.2.8",
     "aos": "^2.3.4",
     "arb-ts": "^1.0.2",
-    "jquery": "^3.6.0",
```

Comment:
HOW?

Reply:
I think that's some leftover from landing page work, I don't even want to ask WHY xD